src/js/core.js

```diff
@@ -520,9 +520,14 @@ function MediumEditor(elements, options) {
                     element.innerHTML = element.innerHTML;
                 }
 
+                // cleanup extra added attributes
                 element.removeAttribute('contentEditable');
                 element.removeAttribute('spellcheck');
+                element.removeAttribute('data-medium-focused');
                 element.removeAttribute('data-medium-editor-element');
+                element.removeAttribute('role');
```

Comment:
I would vote for only removing attributes that we knowingly added. I'm not sure why `role` and `aria-multiline` are getting added by some browsers, but if someone purposefully added those attributes before initializing and destroying MediumEditor, I don't think we should remove them.

So yeah, can we reduce this to only remove `data-medium-focused` and `medium-editor-index`?